opencti-platform/opencti-graphql/src/manager/publisherManager.ts

```diff
@@ -184,7 +183,7 @@ export async function handleSimplifiedEmailNotification(
   const renderedEmail = ejs.render(SIMPLIFIED_EMAIL_TEMPLATE, finalTemplateData);
 
   const emailPayload = {
-    from: `${settings.platform_title} <${settings.platform_email}>`,
+    from: await smtpComputeFrom(settings.platform_title),
```

Comment:
since settings.platform_title is the default, it could be removed and so setting is no more needed in this method